tests/test_classification.py

```diff
@@ -166,7 +166,7 @@ def get_params(self):
         # To achieve testing in a reasonnable amount of time,
         # we will lower the size of the feature and the number of trials
         return {
-            "n_samples": 4,
+            "n_samples": 6,
```

Comment:
`error with vector dimension=4` did not give a clue on whether the problem is with the samples or the features.